base/applications/osk/lang/de-DE.rc

```diff
@@ -162,67 +162,67 @@ BEGIN
     PUSHBUTTON      "8",SCAN_CODE_9,163,22,15,15
     PUSHBUTTON      "9",SCAN_CODE_10,183,22,15,15
     PUSHBUTTON      "0",SCAN_CODE_11,203,22,15,15
-    PUSHBUTTON      "-",SCAN_CODE_12,223,22,15,15
-    PUSHBUTTON      "=",SCAN_CODE_13,243,22,15,15
+    PUSHBUTTON      "ß",SCAN_CODE_12,223,22,15,15
+    PUSHBUTTON      "´",SCAN_CODE_13,243,22,15,15
     PUSHBUTTON      "<--",SCAN_CODE_15,263,22,36,15,BS_ICON
     PUSHBUTTON      "Tab",SCAN_CODE_16,3,41,35,15,BS_ICON
-    PUSHBUTTON      "q",SCAN_CODE_17,42,41,15,15
-    PUSHBUTTON      "w",SCAN_CODE_18,61,41,15,15
-    PUSHBUTTON      "e",SCAN_CODE_19,80,41,15,15
-    PUSHBUTTON      "r",SCAN_CODE_20,99,41,15,15
-    PUSHBUTTON      "t",SCAN_CODE_21,118,41,15,15
-    PUSHBUTTON      "y",SCAN_CODE_22,137,41,15,15
-    PUSHBUTTON      "u",SCAN_CODE_23,156,41,15,15
-    PUSHBUTTON      "i",SCAN_CODE_24,175,41,15,15
-    PUSHBUTTON      "o",SCAN_CODE_25,194,41,15,15
-    PUSHBUTTON      "p",SCAN_CODE_26,213,41,15,15
-    PUSHBUTTON      "[",SCAN_CODE_27,232,41,15,15
-    PUSHBUTTON      "]",SCAN_CODE_28,251,41,15,15
-    PUSHBUTTON      "lock",SCAN_CODE_30,3,60,46,15,BS_ICON
-    PUSHBUTTON      "a",SCAN_CODE_31,53,60,15,15
-    PUSHBUTTON      "s",SCAN_CODE_32,72,60,15,15
-    PUSHBUTTON      "d",SCAN_CODE_33,91,60,15,15
-    PUSHBUTTON      "f",SCAN_CODE_34,110,60,15,15
-    PUSHBUTTON      "g",SCAN_CODE_35,129,60,15,15
-    PUSHBUTTON      "h",SCAN_CODE_36,148,60,15,15
-    PUSHBUTTON      "j",SCAN_CODE_37,167,60,15,15
-    PUSHBUTTON      "k",SCAN_CODE_38,186,60,15,15
-    PUSHBUTTON      "l",SCAN_CODE_39,205,60,15,15
-    PUSHBUTTON      ";",SCAN_CODE_40,224,60,15,15
-    PUSHBUTTON      "'",SCAN_CODE_41,243,60,15,15
-    PUSHBUTTON      "\\",SCAN_CODE_29,270,41,30,15
-    PUSHBUTTON      "ret",SCAN_CODE_43,262,60,38,15,BS_ICON
-    PUSHBUTTON      "shift",SCAN_CODE_44,3,79,57,15,BS_ICON|BS_PUSHLIKE|BS_AUTOCHECKBOX
-    PUSHBUTTON      "z",SCAN_CODE_46,64,79,15,15
-    PUSHBUTTON      "x",SCAN_CODE_47,83,79,15,15
-    PUSHBUTTON      "c",SCAN_CODE_48,102,79,15,15
-    PUSHBUTTON      "v",SCAN_CODE_49,121,79,15,15
-    PUSHBUTTON      "b",SCAN_CODE_50,140,79,15,15
-    PUSHBUTTON      "n",SCAN_CODE_51,159,79,15,15
-    PUSHBUTTON      "m",SCAN_CODE_52,178,79,15,15
+    PUSHBUTTON      "Q",SCAN_CODE_17,42,41,15,15
```

Comment:
Why did you changed letters case? It does not belong to translation, but if you want to change, it should be done in a separate PR.

Reply:
Excellent question!
Actually the "Enhanced keyboard" was already translated with upper cases, while "standard keyboard" was not translated and used still lower cases. Now they both use upper case letters, which, to me, look more appealing and clean.
However when we look at the english resource file we see that both keyboard versions over there use lower case letters instead of upper case letters.

I can revert this. Can I force push on my branch or do you prefer another commit?

We should however decide if we either want to use lower or upper case letters across all translations, to keep things unified. With that we can better ensure that the user experience and application behavior across all translations stays the same. Maybe we should open a discussion issue about that?

Reply:
Revert this change to keep the originality with the English resource of OSK (and if you want for the other resources but in a next PR please). In the future when I plan and have time (I can't promise anything) to re-create On-Screen Keyboard it'll no longer use resources and the upper/lower case conversion will be done programmatically.